packages/primero/src/Adaptor.js

```diff
@@ -222,7 +235,11 @@ export function getCases(query, options, callback) {
           );
 
           if (expandedOptions?.withReferrals) {
+            console.log(
+              `This might take a while, we are fetching ${cases.length} cases referrals`
```

Comment:
I'm not a fan of "this might take a while". It's vague, defensive, a bit apologetic; not a very strong tone to be setting.

Can we just say `Fetching ${cases.length}  referrals...`

Reply:
Okay i agree on `might take a while`. 

But something to be aware is a case id might have more than 1 referrals. So saying `Fetching 2 referrals...` is kinda misleading so maybe `Fetching 2 cases referrals...`?